doc/devops/docker-images.md

```diff
@@ -0,0 +1,55 @@
+# Docker images generated with CI
+
+Table of Contents
+=================
+
+   * [Dockerhub](#dockerhub)
```

Comment:
that top-level item can be removed. In general for the TOC: most links won't work cause the hash/name don't match, it's most simple to have the TOC auto-generated with something like [toctoc](https://github.com/n1k0/toctoc)

Reply:
Thanks for the advice! I will correct.